Modules/Challenges/src/Challenges.Infrastructure/Persistence/IServiceCollectionExtensions.cs

```diff
@@ -46,6 +48,7 @@ public static void AddDatabase(this IServiceCollection services, Action<DbOption
         }
 
         services.AddScoped<IChallengesDbContext, ChallengesDbContext>();
```

Comment:
Delete the now unused `IChallengesDbContext` interface.

Reply:
Not just the registration of the the interface, but also the interface itself. 😑